src/signer.ts

```diff
@@ -109,9 +116,24 @@ export class Signer {
    */
   disconnect = (): void => {
     window.removeEventListener('message', this.onMessageListener);
+    this.#owner = null;
     this.#walletOrigin = null;
   };
 
+  /**
+   * Sets owner to signer, thats allows to communicate with a relying party.
+   * @param {IdentityNotAnonymous} options - The options for the signer owner param.
+   * @returns {void}
+   */
+  setOwner = ({owner}: {owner: IdentityNotAnonymous}): void => {
+    if (nonNullish(this.#owner)) {
+      return;
```

Comment:
IMO we should throw an exception and not silently irgnore it because this might be a misusage.